CriticalMass/ManageFriendsViewController.swift

```diff
@@ -46,9 +46,10 @@ class ManageFriendsViewController: UIViewController, IBConstructable, UITableVie
         tableView.register(cellType: FriendTableViewCell.self)
         tableView.register(cellType: FriendSettingsTableViewCell.self)
         tableView.register(viewType: SettingsTableSectionHeader.self)
-        
-        let tapGestureRecognizer = UITapGestureRecognizer(target: self, action: #selector(handleTap))
```

Comment:
What was this gesture for?

Reply:
It should dismiss the keyboard when tapping anywhere in the table view but it broke the feature where tapping an online friend would focus the friend on the map.